lms/lib/courseware_search/test/test_lms_filter_generator.py

```diff
@@ -139,3 +140,9 @@ def test_excludes_multi_orgs_within(self):
         assert 'org' not in exclude_dictionary
         assert 'org' in field_dictionary
         assert ['TestSite3'] == field_dictionary['org']
+
+    @patch('django.conf.settings.SEARCH_SKIP_SHOW_IN_CATALOG_FILTERING', False)
+    def test_excludes_catalog_visibility(self):
```

Comment:
This tests that the search filters include the new update but I assume there is no way to test the search itself because we don't have search setup in unit testing?

Reply:
Yes, you are right.